configure.ac

```diff
@@ -73,6 +73,7 @@ AC_SUBST([CC])
 # Note: This is present for the flexdll bootstrap where it exposed as the old
 # TOOLPREF variable. It would be better if flexdll where updated to require
 # WINDRES instead.
+AC_SUBST([DIRECT_CPP])
```

Comment:
Hmm, `AC_CHECK_TOOL` already calls `AC_SUBST`, but of course `AC_CHECK_TOOL` is not being called on MSVC. Perhaps it would be better to be explicit further down, and have `AC_SUBST([DIRECT_CPP],[$CPP])` instead of my suggested `DIRECT_CPP="$CPP"`?